packages/sit-onyx/src/components/OnyxSelect/OnyxSelect.vue

```diff
@@ -119,7 +136,26 @@ const { requiredMarkerClass, requiredTypeClass } = useRequired(props);
 </template>
 
 <style lang="scss">
+@use "../../styles/density.scss";
+
+.onyx-select,
+.onyx-select-skeleton {
+  @include density.compact {
+    --onyx-select-padding-vertical: var(--onyx-spacing-4xs);
+  }
+
+  @include density.default {
+    --onyx-select-padding-vertical: var(--onyx-spacing-2xs);
+  }
+
+  @include density.cozy {
+    --onyx-select-padding-vertical: var(--onyx-spacing-sm);
+  }
+}
+
 .onyx-select {
+  $line-height: 1.5rem;
```

Comment:
we could make use of the [lh](https://developer.mozilla.org/en-US/docs/Web/CSS/length#lh) css length unit